LeoYReyes/mmm-common
# mmm-81/vs options parser update

Changes
* add lotto parsing for 'some' and 'small'
* parser support for strike prices that use apostrophe and use the word 'some' before the ticker

Testing
1. verify that the regex looks good
2. verify that the added tests make sense
3. run 'pytest'
